actors/badge/src/main/java/org/sunbird/badge/util/BadgingMessage.java

```diff
@@ -0,0 +1,17 @@
+package org.sunbird.badge.util;
+
+/**
+ * This class will keep all the Messages used inside badging infra. it will have naming convention
```

Comment:
Messages used in badging module. Prefix error messages with "ERROR_" and success messages with "SUCCESS_".